<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CreatePat</class>
 <widget class="QDialog" name="CreatePat">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>438</width>
    <height>520</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="font">
      <font>
       <family>Bahnschrift SemiBold SemiConden</family>
       <pointsize>26</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>PATIENT</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="formWidget" native="true">
     <layout class="QFormLayout" name="formLayout">
      <property name="verticalSpacing">
       <number>20</number>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="idLb">
        <property name="font">
         <font>
          <family>Bahnschrift Light SemiCondensed</family>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Enter ID: </string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="idLn">
        <property name="font">
         <font>
          <family>Bahnschrift SemiLight SemiConde</family>
          <pointsize>14</pointsize>
         </font>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="surnameLb">
        <property name="font">
         <font>
          <family>Bahnschrift Light SemiCondensed</family>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Enter Surname:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="surnameLn">
        <property name="font">
         <font>
          <family>Bahnschrift SemiLight SemiConde</family>
          <pointsize>14</pointsize>
         </font>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="firstNameLb">
        <property name="font">
         <font>
          <family>Bahnschrift Light SemiCondensed</family>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Enter First Name:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="firstNameLn">
        <property name="font">
         <font>
          <family>Bahnschrift SemiLight SemiConde</family>
          <pointsize>14</pointsize>
         </font>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="lastNameILb">
        <property name="font">
         <font>
          <family>Bahnschrift Light SemiCondensed</family>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Enter Last Name: </string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLineEdit" name="lastNameLn">
        <property name="font">
         <font>
          <family>Bahnschrift SemiLight SemiConde</family>
          <pointsize>14</pointsize>
         </font>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="addressLb">
        <property name="font">
         <font>
          <family>Bahnschrift Light SemiCondensed</family>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Enter Address: </string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QLineEdit" name="addressLn">
        <property name="font">
         <font>
          <family>Bahnschrift SemiLight SemiConde</family>
          <pointsize>14</pointsize>
         </font>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="phoneNumberLb">
        <property name="font">
         <font>
          <family>Bahnschrift Light SemiCondensed</family>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Enter Phone Number: </string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QLineEdit" name="phoneNumLn">
        <property name="font">
         <font>
          <family>Bahnschrift SemiLight SemiConde</family>
          <pointsize>14</pointsize>
         </font>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="medNumLb">
        <property name="font">
         <font>
          <family>Bahnschrift Light SemiCondensed</family>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Enter Medical Number: </string>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="QLineEdit" name="medNumLn">
        <property name="font">
         <font>
          <family>Bahnschrift SemiLight SemiConde</family>
          <pointsize>14</pointsize>
         </font>
        </property>
       </widget>
      </item>
      <item row="7" column="0">
       <widget class="QLabel" name="diagnosisLb">
        <property name="font">
         <font>
          <family>Bahnschrift Light SemiCondensed</family>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Enter diagnosis: </string>
        </property>
       </widget>
      </item>
      <item row="7" column="1">
       <widget class="QLineEdit" name="diagnosisLn">
        <property name="font">
         <font>
          <family>Bahnschrift SemiLight SemiConde</family>
          <pointsize>14</pointsize>
         </font>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="confirmPatPb">
     <property name="font">
      <font>
       <family>Bahnschrift SemiBold SemiConden</family>
       <pointsize>20</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>CREATE</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
